docs/tutorials/enable-totp-openstack.md

```diff
@@ -0,0 +1,93 @@
+# Configuring Time-Based One-Time Password (TOTP) on OpenStack
+
+Author: Ramon Grullon
+
+## Introduction
+
+Horizon added time-based one-time password (TOTP) authentication support,
+leveraging the already existing two factor authentication from Keystone.
+Now, if a user activates TOTP on Keystone, it gets activated on Horizon too.
+
+## Prerequesites
+
+- A working OpenStack 2023.2 environment deployed using kolla-ansible.
+- Enabled totp on all control nodes
+- Admin access to the OpenStack dashboard and CLI.
+- Access to the kolla-ansible configuration files.
+
+### Create and Configure a TOTP Credential
+
+On one of your control nodes with OpenStack CLI configured:
+
+Create a project and user to utilize TOTP
```

Comment:
A project is not necessarily needed to enable TOTP. I think explaining how to do this for an existing user makes sense. Additionally when a user is created with openstack cli, flags like `--enable-multi-factor-auth` can be set enabling this from the get-go